man/machine-info.xml

```diff
@@ -128,6 +128,15 @@
         specific as <literal>Left Rack, 2nd Shelf</literal>.
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>KERNEL_INSTALL_MACHINE_ID=</varname></term>
+
+        <listitem><para>Specifies the machine ID <command>kernel-install</command> should use. The value must
```

Comment:
I think it'd be better to "s/machine ID/installation-specific installation directory/". The fact that we sometimes use the machine ID for this is an implementation detail, made even less relevant by this patchset.